# Enforce consistent regex variable name

Place rule description here...

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js
// Place your code here
```

Examples of **correct** code for this rule:

```js
// Place your code here
```

### Options

If there are any options, describe them here. Otherwise, delete this section.
